xapian-letor/ranker/ranker.cc

```diff
@@ -281,22 +291,15 @@ Xapian::prepare_training_file(const string & db_path, const string & queryfile,
 	    throw Xapian::LetorParseError("Query id should be unique");
 	}
 
-	Xapian::Query query_no_prefix = parser.parse_query(querystr,
-					parser.FLAG_DEFAULT|
-					parser.FLAG_SPELLING_CORRECTION);
-	// query with 'title' field as default prefix "S"
-	Xapian::Query query_default_prefix = parser.parse_query(querystr,
-					     parser.FLAG_DEFAULT|
-					     parser.FLAG_SPELLING_CORRECTION,
-					     "S");
-	// Combine queries
-	Xapian::Query query = Xapian::Query(Xapian::Query::OP_OR, query_no_prefix, query_default_prefix);
+	Xapian::Query query = parser.parse_query(querystr);
 
 	Xapian::Enquire enquire(letor_db);
 	enquire.set_query(query);
 	Xapian::MSet mset = enquire.get_mset(0, msetsize);
 
-	std::vector<FeatureVector> fvv_mset = flist.create_feature_vectors(mset, query, letor_db);
+	std::vector<FeatureVector> fvv_mset =
+					flist.create_feature_vectors(mset,
+					query, letor_db, flag, bias);
```

Comment:
This isn't how we'd typically wrap a line.  Especially wrapping the parameters at the same indent as the object the method is called on is likely to make the reader have to stop and and think in a distracting way.

(This isn't really specific to Xapian even - I can't think of a project I've seen where the coding conventions would be to wrap like this!)

Also I notice we're in a file where `using namespace std;` is in use, so the `std::` can be omitted, which helps.

I'd suggest either:

```
        vector<FeatureVector> fvv_mset = flist.create_feature_vectors(mset,
                                                                      query,
                                                                      letor_db,
                                                                      flag,
                                                                      bias);
```

Or:

```
        vector<FeatureVector> fvv_mset =
            flist.create_feature_vectors(mset, query, letor_db, flag, bias);

```